bin/configure

```diff
@@ -1,295 +1,48 @@
 #!/usr/bin/env ruby
 
-puts ""
```

Comment:
The entire diff for this file looks really messy because I extracted various parts of it into sub scripts and changed the order in which things happen. So some of the parts of the diff look like they don't make any sense because it'll look like I removed a block about checking the ruby version and replaced it with a sub script for checking postgres.